2013-02-05 18:13:26,373 - [INFO] - from play in main 
Listening for HTTP on port 9000...

2013-02-05 18:14:48,071 - [ERROR] - from com.jolbox.bonecp.hooks.AbstractConnectionHook in play-akka.actor.default-dispatcher-3 
Failed to obtain initial connection Sleeping for 0ms and trying again. Attempts left: 0. Exception: null

2013-02-05 18:14:48,086 - [ERROR] - from application in New I/O  worker #1 


! @6d8hllg2n - Internal server error, for request [GET /index?username=admin&password=admin123456] ->

play.api.Configuration$$anon$1: Configuration error [Cannot connect to database [default]]
	at play.api.Configuration$.play$api$Configuration$$configError(Configuration.scala:71) ~[play_2.9.1.jar:2.0.3]
	at play.api.Configuration.reportError(Configuration.scala:258) ~[play_2.9.1.jar:2.0.3]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:249) ~[play_2.9.1.jar:2.0.3]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240) ~[play_2.9.1.jar:2.0.3]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.LinearSeqOptimized$class.foreach(LinearSeqOptimized.scala:59) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.foreach(List.scala:45) ~[scala-library.jar:0.11.3]
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.map(List.scala:45) ~[scala-library.jar:0.11.3]
	at play.api.db.BoneCPPlugin.onStart(DB.scala:240) ~[play_2.9.1.jar:2.0.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at scala.collection.LinearSeqOptimized$class.foreach(LinearSeqOptimized.scala:59) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.foreach(List.scala:45) ~[scala-library.jar:0.11.3]
	at play.api.Play$.start(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3$$anonfun$1.apply(ApplicationProvider.scala:125) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3$$anonfun$1.apply(ApplicationProvider.scala:112) ~[play_2.9.1.jar:2.0.3]
	at scala.Option.map(Option.scala:133) ~[scala-library.jar:0.11.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3.apply(ApplicationProvider.scala:112) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at scala.Either$RightProjection.flatMap(Either.scala:277) ~[scala-library.jar:0.11.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at akka.dispatch.Future$$anon$3.liftedTree1$1(Future.scala:195) ~[akka-actor.jar:2.0.2]
	at akka.dispatch.Future$$anon$3.run(Future.scala:194) ~[akka-actor.jar:2.0.2]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:94) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1381) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinTask.doExec(ForkJoinTask.java:259) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinPool.runWorker(ForkJoinPool.java:1479) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104) ~[akka-actor.jar:2.0.2]
Caused by: java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:387) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:414) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:165) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:35) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:801) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at java.sql.DriverManager.getConnection(DriverManager.java:582) ~[na:1.6.0_34]
	at java.sql.DriverManager.getConnection(DriverManager.java:185) ~[na:1.6.0_34]
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:256) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:305) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCPDataSource.maybeInit(BoneCPDataSource.java:150) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:112) ~[bonecp.jar:0.7.1.RELEASE]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:242) ~[play_2.9.1.jar:2.0.3]
	... 29 common frames omitted

2013-02-05 18:14:48,112 - [ERROR] - from application in New I/O  worker #7 


! @6d8hllg2n - Internal server error, for request [GET /icon] ->

play.api.Configuration$$anon$1: Configuration error [Cannot connect to database [default]]
	at play.api.Configuration$.play$api$Configuration$$configError(Configuration.scala:71) ~[play_2.9.1.jar:2.0.3]
	at play.api.Configuration.reportError(Configuration.scala:258) ~[play_2.9.1.jar:2.0.3]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:249) ~[play_2.9.1.jar:2.0.3]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240) ~[play_2.9.1.jar:2.0.3]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.LinearSeqOptimized$class.foreach(LinearSeqOptimized.scala:59) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.foreach(List.scala:45) ~[scala-library.jar:0.11.3]
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.map(List.scala:45) ~[scala-library.jar:0.11.3]
	at play.api.db.BoneCPPlugin.onStart(DB.scala:240) ~[play_2.9.1.jar:2.0.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at scala.collection.LinearSeqOptimized$class.foreach(LinearSeqOptimized.scala:59) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.foreach(List.scala:45) ~[scala-library.jar:0.11.3]
	at play.api.Play$.start(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3$$anonfun$1.apply(ApplicationProvider.scala:125) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3$$anonfun$1.apply(ApplicationProvider.scala:112) ~[play_2.9.1.jar:2.0.3]
	at scala.Option.map(Option.scala:133) ~[scala-library.jar:0.11.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3.apply(ApplicationProvider.scala:112) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at scala.Either$RightProjection.flatMap(Either.scala:277) ~[scala-library.jar:0.11.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at akka.dispatch.Future$$anon$3.liftedTree1$1(Future.scala:195) ~[akka-actor.jar:2.0.2]
	at akka.dispatch.Future$$anon$3.run(Future.scala:194) ~[akka-actor.jar:2.0.2]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:94) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1381) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinTask.doExec(ForkJoinTask.java:259) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinPool.runWorker(ForkJoinPool.java:1479) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104) ~[akka-actor.jar:2.0.2]
Caused by: java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:387) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:414) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:165) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:35) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:801) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at java.sql.DriverManager.getConnection(DriverManager.java:582) ~[na:1.6.0_34]
	at java.sql.DriverManager.getConnection(DriverManager.java:185) ~[na:1.6.0_34]
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:256) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:305) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCPDataSource.maybeInit(BoneCPDataSource.java:150) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:112) ~[bonecp.jar:0.7.1.RELEASE]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:242) ~[play_2.9.1.jar:2.0.3]
	... 29 common frames omitted

2013-02-05 18:14:48,136 - [ERROR] - from application in New I/O  worker #2 


! @6d8hllg2n - Internal server error, for request [GET /icon] ->

play.api.Configuration$$anon$1: Configuration error [Cannot connect to database [default]]
	at play.api.Configuration$.play$api$Configuration$$configError(Configuration.scala:71) ~[play_2.9.1.jar:2.0.3]
	at play.api.Configuration.reportError(Configuration.scala:258) ~[play_2.9.1.jar:2.0.3]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:249) ~[play_2.9.1.jar:2.0.3]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240) ~[play_2.9.1.jar:2.0.3]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.LinearSeqOptimized$class.foreach(LinearSeqOptimized.scala:59) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.foreach(List.scala:45) ~[scala-library.jar:0.11.3]
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.map(List.scala:45) ~[scala-library.jar:0.11.3]
	at play.api.db.BoneCPPlugin.onStart(DB.scala:240) ~[play_2.9.1.jar:2.0.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at scala.collection.LinearSeqOptimized$class.foreach(LinearSeqOptimized.scala:59) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.foreach(List.scala:45) ~[scala-library.jar:0.11.3]
	at play.api.Play$.start(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3$$anonfun$1.apply(ApplicationProvider.scala:125) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3$$anonfun$1.apply(ApplicationProvider.scala:112) ~[play_2.9.1.jar:2.0.3]
	at scala.Option.map(Option.scala:133) ~[scala-library.jar:0.11.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3.apply(ApplicationProvider.scala:112) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at scala.Either$RightProjection.flatMap(Either.scala:277) ~[scala-library.jar:0.11.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at akka.dispatch.Future$$anon$3.liftedTree1$1(Future.scala:195) ~[akka-actor.jar:2.0.2]
	at akka.dispatch.Future$$anon$3.run(Future.scala:194) ~[akka-actor.jar:2.0.2]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:94) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1381) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinTask.doExec(ForkJoinTask.java:259) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinPool.runWorker(ForkJoinPool.java:1479) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104) ~[akka-actor.jar:2.0.2]
Caused by: java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:387) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:414) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:165) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:35) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:801) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at java.sql.DriverManager.getConnection(DriverManager.java:582) ~[na:1.6.0_34]
	at java.sql.DriverManager.getConnection(DriverManager.java:185) ~[na:1.6.0_34]
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:256) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:305) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCPDataSource.maybeInit(BoneCPDataSource.java:150) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:112) ~[bonecp.jar:0.7.1.RELEASE]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:242) ~[play_2.9.1.jar:2.0.3]
	... 29 common frames omitted

2013-02-05 18:15:39,310 - [ERROR] - from application in New I/O  worker #7 


! @6d8hllg2n - Internal server error, for request [GET /index] ->

play.api.Configuration$$anon$1: Configuration error [Cannot connect to database [default]]
	at play.api.Configuration$.play$api$Configuration$$configError(Configuration.scala:71) ~[play_2.9.1.jar:2.0.3]
	at play.api.Configuration.reportError(Configuration.scala:258) ~[play_2.9.1.jar:2.0.3]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:249) ~[play_2.9.1.jar:2.0.3]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240) ~[play_2.9.1.jar:2.0.3]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.LinearSeqOptimized$class.foreach(LinearSeqOptimized.scala:59) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.foreach(List.scala:45) ~[scala-library.jar:0.11.3]
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.map(List.scala:45) ~[scala-library.jar:0.11.3]
	at play.api.db.BoneCPPlugin.onStart(DB.scala:240) ~[play_2.9.1.jar:2.0.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at scala.collection.LinearSeqOptimized$class.foreach(LinearSeqOptimized.scala:59) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.foreach(List.scala:45) ~[scala-library.jar:0.11.3]
	at play.api.Play$.start(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3$$anonfun$1.apply(ApplicationProvider.scala:125) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3$$anonfun$1.apply(ApplicationProvider.scala:112) ~[play_2.9.1.jar:2.0.3]
	at scala.Option.map(Option.scala:133) ~[scala-library.jar:0.11.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3.apply(ApplicationProvider.scala:112) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at scala.Either$RightProjection.flatMap(Either.scala:277) ~[scala-library.jar:0.11.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at akka.dispatch.Future$$anon$3.liftedTree1$1(Future.scala:195) ~[akka-actor.jar:2.0.2]
	at akka.dispatch.Future$$anon$3.run(Future.scala:194) ~[akka-actor.jar:2.0.2]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:94) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1381) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinTask.doExec(ForkJoinTask.java:259) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinPool.runWorker(ForkJoinPool.java:1479) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104) ~[akka-actor.jar:2.0.2]
Caused by: java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:387) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:414) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:165) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:35) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:801) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at java.sql.DriverManager.getConnection(DriverManager.java:582) ~[na:1.6.0_34]
	at java.sql.DriverManager.getConnection(DriverManager.java:185) ~[na:1.6.0_34]
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:256) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:305) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCPDataSource.maybeInit(BoneCPDataSource.java:150) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:112) ~[bonecp.jar:0.7.1.RELEASE]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:242) ~[play_2.9.1.jar:2.0.3]
	... 29 common frames omitted

2013-02-05 18:15:46,089 - [ERROR] - from application in New I/O  worker #7 


! @6d8hllg2n - Internal server error, for request [GET /login] ->

play.api.Configuration$$anon$1: Configuration error [Cannot connect to database [default]]
	at play.api.Configuration$.play$api$Configuration$$configError(Configuration.scala:71) ~[play_2.9.1.jar:2.0.3]
	at play.api.Configuration.reportError(Configuration.scala:258) ~[play_2.9.1.jar:2.0.3]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:249) ~[play_2.9.1.jar:2.0.3]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240) ~[play_2.9.1.jar:2.0.3]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.LinearSeqOptimized$class.foreach(LinearSeqOptimized.scala:59) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.foreach(List.scala:45) ~[scala-library.jar:0.11.3]
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.map(List.scala:45) ~[scala-library.jar:0.11.3]
	at play.api.db.BoneCPPlugin.onStart(DB.scala:240) ~[play_2.9.1.jar:2.0.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at scala.collection.LinearSeqOptimized$class.foreach(LinearSeqOptimized.scala:59) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.foreach(List.scala:45) ~[scala-library.jar:0.11.3]
	at play.api.Play$.start(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3$$anonfun$1.apply(ApplicationProvider.scala:125) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3$$anonfun$1.apply(ApplicationProvider.scala:112) ~[play_2.9.1.jar:2.0.3]
	at scala.Option.map(Option.scala:133) ~[scala-library.jar:0.11.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3.apply(ApplicationProvider.scala:112) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at scala.Either$RightProjection.flatMap(Either.scala:277) ~[scala-library.jar:0.11.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at akka.dispatch.Future$$anon$3.liftedTree1$1(Future.scala:195) ~[akka-actor.jar:2.0.2]
	at akka.dispatch.Future$$anon$3.run(Future.scala:194) ~[akka-actor.jar:2.0.2]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:94) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1381) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinTask.doExec(ForkJoinTask.java:259) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinPool.runWorker(ForkJoinPool.java:1479) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104) ~[akka-actor.jar:2.0.2]
Caused by: java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:387) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:414) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:165) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:35) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:801) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at java.sql.DriverManager.getConnection(DriverManager.java:582) ~[na:1.6.0_34]
	at java.sql.DriverManager.getConnection(DriverManager.java:185) ~[na:1.6.0_34]
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:256) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:305) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCPDataSource.maybeInit(BoneCPDataSource.java:150) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:112) ~[bonecp.jar:0.7.1.RELEASE]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:242) ~[play_2.9.1.jar:2.0.3]
	... 29 common frames omitted

2013-02-05 18:15:48,489 - [ERROR] - from application in New I/O  worker #7 


! @6d8hllg2n - Internal server error, for request [GET /logon] ->

play.api.Configuration$$anon$1: Configuration error [Cannot connect to database [default]]
	at play.api.Configuration$.play$api$Configuration$$configError(Configuration.scala:71) ~[play_2.9.1.jar:2.0.3]
	at play.api.Configuration.reportError(Configuration.scala:258) ~[play_2.9.1.jar:2.0.3]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:249) ~[play_2.9.1.jar:2.0.3]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240) ~[play_2.9.1.jar:2.0.3]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.LinearSeqOptimized$class.foreach(LinearSeqOptimized.scala:59) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.foreach(List.scala:45) ~[scala-library.jar:0.11.3]
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:194) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.map(List.scala:45) ~[scala-library.jar:0.11.3]
	at play.api.db.BoneCPPlugin.onStart(DB.scala:240) ~[play_2.9.1.jar:2.0.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at scala.collection.LinearSeqOptimized$class.foreach(LinearSeqOptimized.scala:59) ~[scala-library.jar:0.11.3]
	at scala.collection.immutable.List.foreach(List.scala:45) ~[scala-library.jar:0.11.3]
	at play.api.Play$.start(Play.scala:60) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3$$anonfun$1.apply(ApplicationProvider.scala:125) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3$$anonfun$1.apply(ApplicationProvider.scala:112) ~[play_2.9.1.jar:2.0.3]
	at scala.Option.map(Option.scala:133) ~[scala-library.jar:0.11.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3.apply(ApplicationProvider.scala:112) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$3.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at scala.Either$RightProjection.flatMap(Either.scala:277) ~[scala-library.jar:0.11.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.9.1.jar:2.0.3]
	at akka.dispatch.Future$$anon$3.liftedTree1$1(Future.scala:195) ~[akka-actor.jar:2.0.2]
	at akka.dispatch.Future$$anon$3.run(Future.scala:194) ~[akka-actor.jar:2.0.2]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:94) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1381) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinTask.doExec(ForkJoinTask.java:259) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinPool.runWorker(ForkJoinPool.java:1479) ~[akka-actor.jar:2.0.2]
	at akka.jsr166y.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104) ~[akka-actor.jar:2.0.2]
Caused by: java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:387) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:414) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:165) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:35) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:801) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.1.0"]
	at java.sql.DriverManager.getConnection(DriverManager.java:582) ~[na:1.6.0_34]
	at java.sql.DriverManager.getConnection(DriverManager.java:185) ~[na:1.6.0_34]
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:256) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:305) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCPDataSource.maybeInit(BoneCPDataSource.java:150) ~[bonecp.jar:0.7.1.RELEASE]
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:112) ~[bonecp.jar:0.7.1.RELEASE]
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:242) ~[play_2.9.1.jar:2.0.3]
	... 29 common frames omitted

